docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     - 80:80
     - 443:443
     - 9200:9200
+    - 9300:9300
```

Comment:
I really don't think we want to open either 9200 or 9300. They should be hidden behind nginx under /search. Also, I think 9300 is only necessary if you have elastic clusters.